mockup/patterns/tinymce/pattern.js

```diff
@@ -310,35 +350,34 @@ define([
           self.tiny = editor;
         }
       };
-      if (tinyOptions.language !== 'en') {
-        tinymce.baseURL = self.options.loadingBaseUrl;
-      }
 
-      if(typeof(self.options.scales) === 'string'){
-        self.options.scales = _.map(self.options.scales.split(','), function(scale){
-          var scale = scale.split(':');
-          return {
-            part: scale[1],
-            name: scale[1],
-            label: scale[0]
-          }
-        });
-      }
-      if(typeof(self.options.folderTypes) === 'string'){
-        self.options.folderTypes = self.options.folderTypes.split(',');
-      }
-      if(typeof(self.options.imageTypes) === 'string'){
-        self.options.imageTypes = self.options.imageTypes.split(',');
-      }
+      self.initLanguage(function() {
+        if(typeof(self.options.scales) === 'string'){
+          self.options.scales = _.map(self.options.scales.split(','), function(scale){
+            var scale = scale.split(':');
+            return {
+              part: scale[1],
+              name: scale[1],
+              label: scale[0]
+            }
+          });
+        }
+        if(typeof(self.options.folderTypes) === 'string'){
```

Comment:
shouldn't that and the following code be outside the current block and not within ``initLanguage``?